parallaxinc/solo
# Solo 200 resubmit-D constant and function names and cleanup

Continues addressing #279 
 - Renames several constants to bring them inline with formatting/style guides
 - Improves comPort getter function to use dropdown's value as opposed to UI selection.
 - Uses class function for focusing the terminal console instead of focusing the HTML element.

To Test:
 - Make sure port menu populates and is able to select and reference a board/device to program.
